SofaKernel/modules/SofaBaseMechanics/MechanicalObject.inl

```diff
@@ -2623,6 +2623,32 @@ void MechanicalObject<DataTypes>::getConstraintJacobian(const core::ExecParams*
 }
 
 template <class DataTypes>
+void MechanicalObject<DataTypes>::buildIdentityBlocksInJacobian(const sofa::helper::vector<unsigned int>& list_n, core::MatrixDerivId &mID)
```

Comment:
To me, it breaks the purpose of mstates to make them build matrices...
To me mstates should only store state vectors, but this idea seems definitively lost.